org-formation/050-costs/program-code-rules.json

```diff
@@ -0,0 +1,399 @@
+[
+  {
+    "Type": "REGULAR",
+    "Value": "Platform Infrastructure",
+    "Rule": {
+      "And": [
+    {
+      "Dimensions": {
+        "Key": "LINKED_ACCOUNT",
+        "Values": [
+          "531805629419",
+          "140124849929",
+          "254940560790",
+          "231505186444",
+          "153370007719",
+          "087547153226",
+          "797640923903",
+          "867686887310",
+          "420786776710",
+          "745159704268",
+          "325565585839",
+          "383874245509",
+          "449435941126",
+          "649232250620",
+          "804034162148",
+          "743644221192"
+        ],
+        "MatchOptions": [
+          "EQUALS"
+        ]
+      }
+    },
+    {
+      "Tags": {
+        "Key": "CostCenter",
+        "MatchOptions": [
+          "ABSENT"
+        ]
+      }
+    },
+    {
+      "Tags": {
+        "Key": "CostCenterOther",
+        "MatchOptions": [
+          "ABSENT"
+        ]
+      }
+    }
+      ]
+    }
+  },
+  {
+    "Type": "REGULAR",
+    "Value": "Platform Infrastructure",
+    "Rule": {
+      "Tags": {
+    "Key": "CostCenter",
+    "Values": [
+      "Platform Infrastructure"
+    ],
+    "MatchOptions": [
+      "STARTS_WITH"
+    ]
+      }
+    }
+  },
+  {
+    "Type": "REGULAR",
+    "Value": "BMGF Ki",
+    "Rule": {
+      "Tags": {
+    "Key": "CostCenter",
+    "Values": [
+      "BMGF-Ki"
+    ],
+    "MatchOptions": [
+      "STARTS_WITH"
+    ]
+      }
+    }
+  },
+  {
+    "Type": "REGULAR",
+    "Value": "BMGF Ki",
+    "Rule": {
+      "Tags": {
+    "Key": "CostCenterOther",
+    "Values": [
+      "BMGF-Ki"
+    ],
+    "MatchOptions": [
+      "STARTS_WITH"
+    ]
+      }
+    }
+  },
+  {
+    "Type": "REGULAR",
+    "Value": "BMGF Ki",
+    "Rule": {
+      "Dimensions": {
+    "Key": "LINKED_ACCOUNT",
+    "Values": [
+      "464102568320"
+    ],
+    "MatchOptions": [
+      "EQUALS"
+    ]
+      }
+    }
+  },
+  {
+    "Type": "REGULAR",
+    "Value": "CD2H CU",
```

Comment:
Shouldn't these values be the full program code strings, e.g. `CD2H-CU / 113300`?

Reply:
The value can only have numbers, letters, or whitespace. This doesn't apply to values inherited from tags, only to values set explicitly in these rules. I stripped off the numeric code when I removed the delimiter, but maybe that was a mistake.